### ISSUE TITLE
The pop-up opens out of the window

### ISSUE THREAD

**Issue by ahmady173** [Kiarash-Z/react-persian-calendar-date-picker#17]
If the component is used at the window sills, the pop-up will opens out of the window

**Comment by thebrodmann** [comment]
@ahmady173 can you provide any repro? maybe a codesandbox?

**Comment by ahmady173** [comment]
@thebrodmann https://codesandbox.io/s/elastic-dijkstra-vryy9

**Comment by Kiarash-Z** [comment]
Hey @ahmady173,
Thanks for submitting this PR. I reviewed a couple of date pickers and they had a similar behavior. What's your ideal behavior for the picker in these situations?

**Comment by thebrodmann** [comment]
In fact, all popups should adjust by the position on the window (e.g. dropdown).
The following adjustments should be considered for all popups (e.g. date picker):

- If popup box **goes beyond** the **left side** of the window it **should be adjusted** to the **right**.
- If popup box **goes beyond** the **right side** of the window it **should be adjusted** to the **left**.
- If popup box **goes beyond** the **bottom side** of the window it **should be adjusted** to the **top**.

**Comment by Kiarash-Z** [comment]
@thebrodmann totally agreed. I will definitely consider this issue in the next release

**Comment by thebrodmann** [comment]
@Kiarash-Z this sounds like bugfix for me. Maybe it would be better to release it under a patch version.

**Comment by Kiarash-Z** [comment]
@thebrodmann Yeah, of course, it will be under a patch version. I will work it out soon

**Comment by mmahdigh** [comment]
@ahmady173 I think if you change `position` property of `.DatePicker__calendarContainer` (in DatePicker.css, unfortunately I didn't find a prop to override this element stylings) from `'absolute'` to `'relative'` then this problem will be fixed.

**Comment by Kiarash-Z** [comment]
@ahmady173 flags got corrected😊

**Comment by Kiarash-Z** [comment]
@mmahdigh That wouldn't fix the problem. the position of the calendar container gotta be `absolute` for the correct spatial behavior.

**Comment by mmahdigh** [comment]
@Kiarash-Z  You're right. but I think having the ability to override .`DatePicker__calendarContainer` css would solve a wide range of problems.

**Comment by Kiarash-Z** [comment]
@mmahdigh You can indirectly do that if you wish. Consider this example plz:
```
<DatePicker
    ...
    wrapperClassName="DatePickerWrapper" {/* adding a class to the wrapper */
/>
```
CSS file:
```
.DatePickerWrapper .DatePicker__calendarContainer { /* selecting calendarContainer based on that */
  position: relative;
}

```

**Comment by thebrodmann** [comment]
> but I think having the ability to override .DatePicker__calendarContainer css would solve a wide range of problems.

@mmahdigh It doesn't help anyone to introduce a solution for a **wide range of problems** which have not any issue! Please feel free to open issue for any problem you think there is and then introduce your solutions. In this way, the **community** can bring their **comment and solutions** to you and the project.

**Comment by Kiarash-Z** [comment]
@thebrodmann @mmahdigh @ahmady173 This is now fixed in the new version(1.1.3). Here are the screenshots:
![image](https://user-images.githubusercontent.com/20098648/62775421-d3b54100-babc-11e9-8e66-7df9bd9fd489.png)
![image](https://user-images.githubusercontent.com/20098648/62775426-d6b03180-babc-11e9-9074-2ee644fa8b28.png)

**Comment by ahmady173** [comment]
great job.
